docs/build/isc/v1.1/docs/tutorials/defi-lend-borrow-tutorial-part-1.md

```diff
@@ -0,0 +1,86 @@
+# DeFi Lend Borrow
+
+DeFi Lend Borrow is a decentralized finance (DeFi) application that enables users to lend and borrow assets on the Shimmer EVM testnet. The project is built using Solidity and Hardhat, with the core functionality provided by smart contracts.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Contracts Overview](#contracts-overview)
+  - [IToken](#itoken)
+  - [InterestRateModel](#interestratemodel)
+  - [ITokenManager](#itokenmanager)
+  - [MockERC20](#mockerc20)
+- [Usage](#usage)
+- [Testing](#testing)
+- [Deployment](#deployment)
+- [License](#license)
+
+## Prerequisites
+
+Before you begin, ensure you have met the following requirements:
+
+- Node.js and npm installed
+- Hardhat installed globally
+- A Shimmer EVM testnet account with test tokens
+
+## Installation
+
+1. Clone the repository:
+    ```bash
+    git clone https://github.com/yourusername/defi-lend-borrow.git
+    cd defi-lend-borrow
+    ```
+
+2. Install the dependencies:
+    ```bash
+    npm install
+    ```
+
+3. Compile the contracts:
+    ```bash
+    npx hardhat compile
+    ```
+
+## Contracts Overview
```

Comment:
If there are multiple contracts, I highly recommend having an architecture diagram to show the user the overall picture.

Reply:
Sure, will integrate it as well, Thanks for the suggestion.